contracts/SnappAuction.sol

```diff
@@ -11,17 +11,11 @@ contract SnappAuction is SnappBase {
     
     struct StandingOrderBatch {
         bytes32 orderHash;
-        uint validFromAuctionIndex; // validity is inclusive of the auction index
         uint validToAuctionIndex; // validity is inclusive of the auction index
     }
 
-    struct StandingOrderData {
-        mapping(uint => StandingOrderBatch) reservedAccountOrders;
-        uint standingOrderNonce;
-    }
-
-    //mapping from accountId to nonce to StandingOrderBatch
-    mapping (uint16 => StandingOrderData) public standingOrderPlacements;
+    //mapping from accountId to validFromAuctionIndex to StandingOrderBatch
```

Comment:
can't this just be standing orders? I don't see the reason for the word "Placement" here